plain-vanilla-games/PVGTableViewProxy
# Minor changes to workspace for nicer pod installation

This PR addresses some weirdness (disappearing code folder) with Cocoapods and the xcworkspace when cloning the project for the first time.

It basically doesn't do much except to slightly change the podfile for the example project to make it use the same xcworkspace and then run pod install.

How to test:
+ Clone a fresh version of the project and check out this branch.
+ Verify that the workspace has references to `PVGTableViewProxy`, `PVGTableViewProxyTests`, `PVGTableViewProxyExample`, and two missing Pod project references.
+ Run `cd Test && pod install` and verify that the workspace still has the same structure and that the tests can be run.
+ Run `cd Example && pod install` and verify that the workspace still has the same structure and that the tests can be run.
